mona_sdk/authentication.py

```diff
@@ -255,13 +273,18 @@ def _calculate_and_set_time_to_refresh(api_key):
         )
 
 
-def _handle_authentications_error(error_message):
+def _handle_authentications_error(error_message, should_raise_exception=None):
     """
     Logs an error and raises MonaAuthenticationException if
     RAISE_AUTHENTICATION_EXCEPTIONS is true, else returns false.
     """
+    should_raise_exception = (
+        should_raise_exception
+        if should_raise_exception is not None
+        else RAISE_AUTHENTICATION_EXCEPTIONS
```

Comment:
I'd also prefer to pass the default here into the function instead of None, any reason not to?

Reply:
@nemo-mona same as my last comment